Makefile

```diff
@@ -66,6 +66,9 @@ test-ffi:
 test-jni: build-jni
 	@( cd java ; ./gradlew test --rerun-tasks)
 
+test-java-lib:
+	@( cd java-lib-test ; ./gradlew test --rerun-tasks)
```

Comment:
Can we move `java-lib-test` inside `java`? It doesn't have to be included inside upper gradle since it is just a small test client